pages/docs/index.mdx

```diff
@@ -1,207 +1,65 @@
+## Du möchtest den digitalen Puls Deiner Stadt fühlen?
+Dann bist Du hier genau richtig! Die Stadt Berlin, Hochschulen aber auch Privatpersonen erheben bereits enorm viele Daten über unsere Stadt. Dabei spielen Klimadaten, aber auch städtische Sensordaten zu Themen wie "Lärmbelästigung" oder "Füllstände von Mülleimern" eine wichtige Rolle.
+Diese Daten in einer gemeinsamen Datenbasis vorzuhalten, zugänglich zu machen und zu visualisieren ist ein erster Schritt in Richtung *smarte vernetzte Stadt* und legt den Grundstein für weitere Analysen, Prozessoptimiereung und für eine bessere Stadtplanung.
+Man könnte sagen:
+<blockquote>
+  <p>Sensordaten legen den Grundstein für eine smarte Stadt der Zukunft.</p>
+</blockquote>
 
-- **It's not a bad idea to add a third item either.**
+Mit Klick auf die <a href='/projects'>Projekte</a> gelangst Du zur Projektübersicht. Hier werden alle Projekte unserer Community gelistet.
+Du siehst auf einem Blick Namen, Kurzbeschreibung, Eigentümer und weitere wichtige Informationen zum Projekt.
 
-  I think it probably would've been fine to just use two items but three is definitely not worse, and since I seem to be having no trouble making up arbitrary things to type, I might as well include it.
+## Du hast bereits ein IoT-Projekt?
+Wunderbar! Erstelle  Dir einen Account und stelle Deine Daten für andere Nutzer:innen und die Öffentlichkeit zur Verfügung. Die Plattform unterstützt bislang die Anbindung an das <a href="https://www.thethingsnetwork.org/" target="_blank">The Things Network (TTN)</a>, sodass Du Sensoren Die Du bereits in TTN registriert hast einfach über die <a href="https://www.thethingsnetwork.org/docs/applications/http/" target="_blank">HTTP-Integration von TTN</a> mit dem IoT Hub verknüpfen kannst.
+Nach erfolgreicher Verknüpfung wird für Deine Sensoren automatisch ein Projekt generiert und die Daten Deiner Sensoren zum Download, sowie über eine REST-API <b>frei als Open Data zur Verfügung gestellt</b>.
 
-After this sort of list I usually have a closing statement or paragraph, because it kinda looks weird jumping right to a heading.
+<Button><a href='/signup'>Jetzt Account anlegen</a></Button>
 
-## Code should look okay by default.
+Im linken Menü findest du unter dem Punkt *Dokumentation* eine ausführliche Erläuterung dazu, wie Du Dir einen Account anlegst oder Deine Sensoren aus TTN mit dem IoT Hub verknüpfst.
 
-I think most people are going to use [highlight.js](https://highlightjs.org/) or [Prism](https://prismjs.com/) or something if they want to style their code blocks but it wouldn't hurt to make them look _okay_ out of the box, even with no syntax highlighting.
+## Du brauchst Daten für Deine Analyse?
 
-Here's what a default `tailwind.config.js` file looks like at the time of writing:
+Dann browse durch <a href='/projects'>Projekte</a> finde den passenden Datensatz für Deine Datenanalyse.
+Bislang unterstützen wir Sensordaten aus <b>sieben verschiedenen Kategorien</b>. Dazu zählen:
```

Comment:
Just a little hint: It's probably better to use `<strong>` here because it semtically indicates importance, not just "bold" as a visual feature.